float32 N
float32 NW
float32 W
float32 SW
float32 S
float32 SE
float32 E
float32 NE
